Description: Fixes the invocation of the exec plugin during the build
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/jmock/pom.xml
+++ b/jmock/pom.xml
@@ -84,8 +84,11 @@
                 </executions>
                 <configuration>
                     <executable>java</executable>
-                    <commandlineArgs>-cp %classpath:target/classes
-                        org.jmock.ExpectationsCreator</commandlineArgs>
+                    <arguments>
+                        <argument>-cp</argument>
+                        <argument>/usr/share/java/asm-all.jar:target/classes</argument>
+                        <argument>org.jmock.ExpectationsCreator</argument>
+                    </arguments>
                 </configuration>
             </plugin>
 
